packages/utilities/test/get-examples-for-schema.test.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright 2017 - 2023 IBM Corporation.
+ * SPDX-License-Identifier: Apache2.0
+ */
+
+const { getExamplesForSchema } = require('../src');
+
+describe('Utility function: getExamplesForSchema()', () => {
+  it('should return `[]` for `undefined`', async () => {
+    expect(getExamplesForSchema(undefined)).toEqual([]);
+  });
+
+  it('should return `[]` for `null`', async () => {
+    expect(getExamplesForSchema(null)).toEqual([]);
+  });
+
+  it('should return `[]` for empty object', async () => {
+    expect(getExamplesForSchema({})).toEqual([]);
+  });
+
+  it('should return examples for simple schema', async () => {
+    expect(
+      getExamplesForSchema({
+        examples: ['three', 'two', 'one'],
+      }).sort()
+    ).toEqual(['three', 'two', 'one'].sort());
```

Comment:
I wouldn't think that `sort()` would be necessary for a list of primitives - am I missing something?

Reply:
Technically, there are no guarantees on the order examples are returned in.